scripts/install/createTables.php

```diff
@@ -58,14 +58,15 @@ public function generateTables(\common_persistence_SqlPersistence $persistence)
             $tableResults->setPrimaryKey(array(RdsResultStorage::RESULTS_TABLE_ID));
 
 
-            $tableVariables->addColumn(RdsResultStorage::VARIABLES_TABLE_ID, "integer", array("autoincrement" => true));
+            $tableVariables->addColumn(RdsResultStorage::VARIABLES_TABLE_ID, 'string', ['length' => 25]);
```

Comment:
uniqid('', true) length = 23, maybe we can spare those 2 ?

Reply:
Fixed with 23 bytes.